aitelemetry/telemetrywrapper.go

```diff
@@ -207,6 +209,8 @@ func (th *telemetryHandle) TrackLog(report Report) {
 	trace.Tags.User().SetAuthUserId(runtime.GOOS)
 	trace.Tags.Operation().SetId(report.Context)
 	trace.Tags.Operation().SetParentId(th.appVersion)
+	trace.Tags.Application().SetVer(th.appVersion)
+	trace.Properties[hostNameKey], _ = os.Hostname()
```

Comment:
is this different from metadata.VMName?

Reply:
yes - AI queries IMDS for some metadata like the "VMName", so you will actually get the VM Name. But the hostname that the OS reports may vary from the vm name and has no connection. Crucially for k8s, the hostname is the Pod name, and we will be able to tell what _Pod_ the logs are coming from, instead of just which VM.